contracts/BSCValidatorSet.sol

```diff
@@ -107,10 +107,11 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
   }
 
   modifier initValidatorExtraSet() {
-    if (validatorExtraSet.length == 0) {
+    uint256 len = currentValidatorSet.length;
```

Comment:
Please do not use `len`, in some other language it is a native function.

Reply:
fixed